packages/core/src/layout/Sidebar/Sidebar.stories.tsx

```diff
@@ -22,14 +22,22 @@ import {
   SidebarDivider,
   SidebarSearchField,
   SidebarSpace,
-  SidebarUserSettings,
-  ProviderSettingsItem,
 } from '.';
 import HomeOutlinedIcon from '@material-ui/icons/HomeOutlined';
 import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
-import Star from '@material-ui/icons/Star';
 import { MemoryRouter } from 'react-router-dom';
-import { githubAuthApiRef } from '@backstage/core-api';
+import {
+  ApiProvider,
+  ApiRegistry,
+  appThemeApiRef,
+  AppThemeSelector,
+  configApiRef,
+  ConfigReader,
+  FeatureFlags,
+  featureFlagsApiRef,
+} from '@backstage/core-api';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Don't think you need this as it is being used

Reply:
@benjdlambert This was to prevent adding the plugin as a dependency to `@backstage/core`... Not sure how that works with storybook but locally it works. Any thoughts on how to handle this?

Reply:
Aha, now I see. Wonder if we should move the story to the `UserSettings` plugin anyway. as it might be it's own page in the future not part of the sidebar?

Reply:
Ah, 👍  I moved that part of the storybook into this plugin. Thanks!